src/Components/ElementsOfSite/AllStories/AllStories.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+interface IAllStoriesProps {
+  withTitle: boolean;
+  SmallCard: React.ReactNode;
+}
+
+const AllStories: React.FC<IAllStoriesProps> = ({ withTitle, SmallCard }) => {
+  return (
+    <div className="container">
+      <section className="recent-posts">
+        {withTitle ? (
+          <div className="section-title">
+            <h2>
+              <span>All Stories</span>
+            </h2>
+          </div>
+        ) : null}
+        <div className="card-columns listrecent">
+          {SmallCard}
+          {SmallCard}
+          {SmallCard}
+          {SmallCard}
+          {SmallCard}
+          {SmallCard}
```

Comment:
Данные для карточек должны храниться в структуре данных, а сами карточки выводиться через метод map, и получать данные через пропсы

```suggestion
          {articles.map((article) => (
              <SmallCard title={article.title} text={article.text} date={article.date}/>
          )